sql/core/src/main/scala/org/apache/spark/sql/SQLConf.scala

```diff
@@ -36,6 +36,8 @@ private[spark] object SQLConf {
   val PARQUET_INT96_AS_TIMESTAMP = "spark.sql.parquet.int96AsTimestamp"
   val PARQUET_CACHE_METADATA = "spark.sql.parquet.cacheMetadata"
   val PARQUET_COMPRESSION = "spark.sql.parquet.compression.codec"
+  val PARQUET_BLOCK_SIZE = "spark.sql.parquet.blocksize"
+  val PARQUET_PAGE_SIZE = "spark.sql.parquet.pagesize"
```

Comment:
@yhuai , I just confirmed, it still keep uncompressed parquet file even if we set the property like `SET parquet.compression=GZIP`.

From the source code, https://github.com/chenghao-intel/spark/blob/parquet/sql/core/src/main/scala/org/apache/spark/sql/parquet/newParquet.scala#L636
Settings in `SQLConf` are not append into the `Configuration`, that's why all of the settings doesn't take effect.